docs/src/metadata/bibliography.yml

```
@@ -47,3 +47,16 @@ monet:
+  type: web
+  title: "Grease: A minimal Monero Payment Channel implementation for Monero"
+  author:
+    - CJ
+    - Sean Coughlin
+  url: https://cfp.twed.org/mk5/talk/QYDGPM/
+jeffro25:
+  type: misc
+  author:
+    - jeffro256
+  title: "Personal communication: Signature changes post-FCMP"
+  date: 2025-06-22
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)